spec/index.bs

```diff
@@ -842,14 +842,24 @@ In this specification, the following syntactic shorthands are used:
     A WebIDL attribute which is backed by an internal slot of the same name.
     It may or may not be mutable.
 
-### WebGPU Interfaces ### {#webgpu-interfaces}
+### WebGPU Objects ### {#webgpu-objects}
```

Comment:
It _probably_ doesn't matter but let's keep a permalink for `webgpu-internal-objects`:
```suggestion
<h4 id=webgpu-objects>WebGPU Objects
<span id=webgpu-internal-objects></span>
</h4>
```